developer-docs-site/docs/concepts/coin-and-token/token_airdrop/Token Airdrop Example c4d35e65f7314699a5335fb96d2d5c1f.md

```diff
@@ -0,0 +1,29 @@
+# Token Airdrop Example
+
+We support different ways of transferring token in our token standard [[link](https://aptos.dev/concepts/coin-and-token/aptos-token#token-transfer)]
```

Comment:
best two define the two methods and then explain them.

Reply:
but you only discuss one approach in this doc

Reply:
This is to show a basic example of doing an airdrop.  I only think of a way to use the 2-step process, which is the reason I only discuss one approach.  The other transfer mode doesn't apply here since we don't know if the whitelist address have opt-in directly transfer